packages/owlbot-bootstrapper/common-container/utils.ts

```diff
@@ -131,21 +172,24 @@ export async function openAnIssue(
 }
 
 /**
- * Gets the branch name with a UUID
+ * Gets variables saved to a well-known file passed between containers
  *
  * @param directoryPath name of the directory in which the process is running (i.e., 'workspace' for a container)
+ * @param fileNameJSON name of the file that contains the inter-container variables, must be JSON
  * @returns the branch name with a UUID from a well-known file path (/workspace/branchName.md)
  * Since a separate container needs the UUID for the branch of the first container,
  * we need to save it in a well-known location that language-specific containers can access later on.
  */
-export async function getBranchName(directoryPath: string) {
+export function getWellKnownFileContents(
+  directoryPath: string,
+  fileNameJSON: string
+): InterContainerVars {
   try {
-    return (
-      fs
-        .readFileSync(`${directoryPath}/${BRANCH_NAME_FILE}`)
-        .toString()
-        // Need to remove whitespace from branch name
-        .replace(/\s+/g, '')
+    console.log(
+      fs.readFileSync(`${directoryPath}/${fileNameJSON}`).toString('utf8')
+    );
+    return JSON.parse(
+      fs.readFileSync(`${directoryPath}/${fileNameJSON}`).toString()
```

Comment:
Might be worth considering setting default values in case the file is not present (not necessary in this PR)